### ISSUE TITLE
wind direction of 350 leads to exception

### ISSUE THREAD

**Issue by BradWalker** [mivek/MetarParser#233]
**Describe the bug**
The following METAR leads to an array out of bounds exception..

"KARB 301153Z 35022G31KT 10SM -RA BKN017 BKN025 OVC039 02/M01 A2948";

```
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 16 out of bounds for length 16
	at io.github.mivek.utils.Converter.degreesToDirection(Converter.java:39)
	at io.github.mivek.command.common.BaseWindCommand.setWindElements(BaseWindCommand.java:22)
	at io.github.mivek.command.common.WindCommand.parseWind(WindCommand.java:32)
	at io.github.mivek.command.common.WindCommand.execute(WindCommand.java:38)
	at io.github.mivek.parser.AbstractParser.generalParse(AbstractParser.java:137)
	at io.github.mivek.parser.MetarParser.parse(MetarParser.java:82)
	at io.github.mivek.parser.MetarParser.parse(MetarParser.java:23)
	at io.github.mivek.service.MetarService.decode(MetarService.java:37)
	at javaapplication1.JavaApplication1.main(JavaApplication1.java:36)
/home/bwalker/.cache/netbeans/11.3/executor-snippets/debug.xml:150: The following error occurred while executing this line:
/home/bwalker/.cache/netbeans/11.3/executor-snippets/debug.xml:129: The following error occurred while executing this line:
/home/bwalker/.cache/netbeans/11.3/executor-snippets/debug.xml:105: Java returned: 1
BUILD FAILED (total time: 1 minute 49 seconds)
```

**To Reproduce**
```
package javaapplication1;

import io.github.mivek.exception.ParseException;
import io.github.mivek.model.Metar;
import io.github.mivek.service.MetarService;
import java.util.logging.Level;
import java.util.logging.Logger;

public class JavaApplication1 {
    public static void main(String[] args) {
        String code = "KARB 301153Z 35022G31KT 10SM -RA BKN017 BKN025 OVC039 02/M01 A2948";
                
        MetarService service = MetarService.getInstance();

        Metar metar = null;

        try {
            metar = service.decode(code);
        } catch (ParseException ex) {
            Logger.getLogger(JavaApplication1.class.getName()).log(Level.SEVERE, null, ex);
        }
 
        System.out.println(metar);
    }    
}
```

**Expected behavior**
Should not fail and return the NNW index in Converter.DIRECTIONS.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Unix
 - Browser Firefox

**Additional context**
nothing to add

**Comment by mivek** [comment]
Hello @BradWalker 
There is indeed an error when converting the direction to a cardinal direction.
I will fix it this weekend. However 350° should return North direction not NNW
http://snowfence.umn.edu/Components/winddirectionanddegrees.htm

**Comment by BradWalker** [comment]
I agree.. I think the simple fix is the following:

```
private static final String[] DIRECTIONS = {"N", "NNE", "NE", "ENE", "E", "ESE", "SE", "SSE",
            "S", "SSW", "SW", "WSW", "W", "WNW", "NW", "NNW", "N"};
```

At the end of the day, both 0 & 360 degrees should return "N".

**Comment by mivek** [comment]
Hello @BradWalker 

I merged the PR fixing the issue.
The fix is available in the version `2.1.2` of the project.

**Comment by BradWalker** [comment]
Hey @mivek, kudos to you for the very prompt response on this bug..
